### DISCUSSION TITLE
more permissive license

### DISCUSSION THREAD

**Discussion by chappedhands** [seung-lab/connected-components-3d#88] [Q&A] [answered]
Could cc3d be licensed with something more permissive than GPL3? Maybe a weak copyleft license?

**Comment by william-silversmith** [comment, marked as answer]
I would consider issuing additional individual licenses for non-profit applications such as research and medicine (possibly other uses) if I think they are publicly beneficial (though I haven't thought through exactly how that would work). I'm not currently considering making the software able to be integrated into non-free distributed applications more generally. For corporate users, I advise them to pay a developer if it's business critical.
